server/errors.json

```diff
@@ -1058,5 +1058,15 @@
     "help": "",
     "url": "",
     "deprecates": ""
+  },
+  {
+    "constant": "JSConsumerWithFlowControlNeedsHeartbeats",
+    "code": 400,
+    "error_code": 10108,
+    "description": "consumer with flow control also needs heartbeats",
+    "comment": "",
+    "help": "",
+    "url": "",
+    "deprecates": ""
   }
-]
+]
```

Comment:
don't know if the team is picky about the blank line at the end of file

Reply:
This is done by the `nats errors add` and in general git will report if there is no newline after the last character, so that's a good thing.